packages/x-components/src/x-modules/search-box/store/actions/set-query-from-url.action.ts

```diff
@@ -0,0 +1,11 @@
+import { SearchBoxXStoreModule } from '../types';
+
+export const setQueryFromUrl: SearchBoxXStoreModule['actions']['setQueryFromUrl'] = (
```

Comment:
I know this action is pretty simple but maybe we should add a documentation here as well.

Reply:
Sure!